common/pom.xml

```diff
@@ -74,6 +80,21 @@
         </dependency>
 
         <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+        </dependency>
```

Comment:
similar question here... can we remove any dependencies in client, now that we have push some code into common?